test/unit/option_spec.lua

```diff
@@ -0,0 +1,64 @@
+
+local helpers = require("test.unit.helpers")
+
+local to_cstr = helpers.to_cstr
+local eq      = helpers.eq
+
+local option = helpers.cimport("./src/nvim/option.h")
+local globals = helpers.cimport("./src/nvim/globals.h")
+
+describe('option functions', function()
+  local check_ff_value = function(ff)
+    return option.check_ff_value(to_cstr(ff))
+  end
+
+  describe('check_ff_value string', function()
+
+    it('should view empty string as valid', function()
+      eq(1, check_ff_value(""))
+    end)
+
+    it('should view "unix" as valid', function()
+      eq(1, check_ff_value("unix"))
+    end)
+    
+    it('should view "dos" as valid', function()
+      eq(1, check_ff_value("dos"))
+    end)
+
+    it('should view "mac" as valid', function()
+      eq(1, check_ff_value("mac"))
+    end)
+
+    it('should view "foo" as invalid', function()
+      eq(0, check_ff_value("foo"))
+    end)
+  end)
+
+  local get_sts_value = function()
+    return option.get_sts_value()
+  end
```

Comment:
might want to group this right next to the other function definition (check_ff_value)